JSQMessagesViewController/Controllers/JSQMessagesViewController.m

```diff
@@ -368,6 +374,7 @@ - (void)finishReceivingMessageAnimated:(BOOL)animated {
 
 - (void)scrollToBottomAnimated:(BOOL)animated
 {
+    [self.collectionView reloadData];
```

Comment:
Without reloading collection view, it keeps old data from previous chat and does not scroll to correct position